internal/configs/virtualserver.go

```diff
@@ -401,14 +419,14 @@ func generateRulesRouteConfig(route conf_v1alpha1.Route, upstreamNamer *upstream
 	}
 }
 
-var specialMapParameters = map[string]bool{
-	"default":   true,
-	"hostnames": true,
-	"include":   true,
-	"volatile":  true,
-}
-
 func generateValueForRulesRouteMap(matchedValue string) (value string, isNegative bool) {
+	var specialMapParameters = map[string]bool{
```

Comment:
this map is a constant. there is no need to put in the `generateValueForRulesRouteMap` func